frontend/src/main/components/Commons/PagedAnnouncementTable.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import OurTable from "main/components/OurTable";
+import { useBackend } from "main/utils/useBackend";
+import { useParams } from "react-router-dom";
+import { timestampToDate } from "main/utils/dateUtils";
+
+
+const PagedAnnouncementTable = () => {
+
+    const refreshJobsIntervalMilliseconds = 5000;
+
+    const { commonsId } = useParams();
+
+    // Stryker disable all
```

Comment:
I use PagedProfitsTable.js as example, and in PagedProfitsTable.js, it has these two `//Stryker` comment, so I also leave them here, I do not know if I can do so, please let me know if I need to write test